modules/compliance-increasing-operator-limits.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-increasing-operator-limits"]
+= Increasing Compliance Operator resource limits
+
+If the Operator has been installed through OLM, the Compliance Operator might require more memory than the default limits allow. The best way to mitigate this issue is to set custom resource limits.
+
+. To increase the Operator's memory limits to 500Mi, create the following patch file:
++
+[source,yaml]
+----
+spec:
+  config:
+    resources:
+      limits:
+        memory: 500Mi
+----
+
+. Apply the patch file:
++
+[source,terminal]
+----
+$ oc patch sub compliance-operator -nopenshift-compliance --patch-file co-memlimit-patch.yaml --type=merge
+----
++
+[IMPORTANT]
+====
+The `nodeselector` field only sets the limit for the compliance-operator
```

Comment:
nodeselector?

Reply:
in addition could we cross link from here to [modules/compliance-scansetting-cr.adoc](https://github.com/openshift/openshift-docs/pull/50778/commits/10394566780a77840128b676759ae92e7c3ce349#diff-075c93655f955ab40ddb30fba38650fd721c60b7e0f3ac04447dafba58600a1b)
we already have a link from there to here, which is awesome